addons/website_mail_channel/models/mail_channel.py

```diff
@@ -45,8 +45,11 @@ def _send_confirmation_email(self, partner_ids, unsubscribe=False):
                 force_send=True,
                 email_values={
                     'recipient_ids': [(4, partner_id)],
-                    'email_from': website.company_id.email,
-                }
+                    'email_from': website.company_id.email_formatted,
+                    'author_id': self.create_uid.partner_id.id,
+                    'message_type': 'user_notification',
+                },
+                notif_layout='mail.mail_notification_light',
```

Comment:
Same question, I don't get this diff in the context of this PR.

Reply:
Globally unrelated change, following Dit testing. I fixed some issues with website_mail_channel not really related to this PR but while I was at it, I fixed it.